### ISSUE TITLE
Stuttering visual

### ISSUE THREAD

**Issue by SkippTekk** [jackun/TestAMFVFW#5]
Hey man, im a tester for your Encoder, Freaking LOVE it. but the only thing that i really have an issue is trying to figure it out, i have knoweldge in this base area but for some reason i can't find any decent settings. i have a R9 390 8gb GPU from AMD (hence why im using this) and my video just stutters and goes into big squares. any idea on a default setting to try out?

**Comment by jackun** [comment]
Try out "Constrained QP" and set slider to 20. For stutter, for some reason it doesn't want to go much above 30 fps at 1080p so try lower fps for a test. Otherwise, I had weird 5-6 fps-only capture issue with Batman: Arkham Knight and Dxtory where enabling "Multi device processing" fixed it more or less. But also good to try if stutter is fixed when using another codec altogether.

**Comment by SkippTekk** [comment]
ok man, i have the settings set. does this work for it? https://i.gyazo.com/196a668ce167901e9430641ebce69d24.png

**Comment by jackun** [comment]
It should. Set preset to Balanced for better quality, should be fast enough still.

**Comment by SkippTekk** [comment]
ok new settings been changed. 

https://i.gyazo.com/4143397ddccf4acd5231487bbc4ae9c4.png doing a testing video now.

Em i your test subject now :3

**Comment by jackun** [comment]
60 FPS is probably too much (for now anyway). Try 48 (if dxtory even allowed that) or even 30 :P

**Comment by SkippTekk** [comment]
but i wanna have 60fps video's on youtube xD

**Comment by jackun** [comment]
720p is probably doable, but there's some weird "lag" with the AMF library that encoding overhead time gets too long for 1080p@60 :( Something is not inited correctly maybe, eh.

**Comment by SkippTekk** [comment]
yeah man that really sucks. i turned it down from 60 to 30. the footage looked WAY better. 
![photo1](https://cloud.githubusercontent.com/assets/6195158/11179236/3a1974a6-8c0f-11e5-8bac-da69384ab6d7.jpg)
that photo was at 60fps 

and now the video of the 30fps.

https://www.youtube.com/watch?v=8qG0UjqgGiE&feature=youtu.be

**Comment by SkippTekk** [comment]
oh and here is the config of your and dxtory. https://i.gyazo.com/ec0f3c84f9ed41ac8cba596efd37a827.png

**Comment by jackun** [comment]
There's still some weird corruption going on. I've seen it when I OCed my GPU a bit. If you have factory OCed card, try downclocking it to ~1000MHz core / 1200MHz mem.

**Comment by SkippTekk** [comment]
it's at 1100mz and im using the MSI Gaming App https://i.gyazo.com/20ce69815cba23c7dbf9f70b090d2dac.png it might be outdated though. ill see if it's needed to be updated

**Comment by jackun** [comment]
Yeah, downclock it a little. Memory clock should have bigger effect though probably and memory corruption probably manifests better as H264 bitstream corruption than pink textures.

**Comment by SkippTekk** [comment]
recorded new session, rendering now with Sony Vegas

**Comment by SkippTekk** [comment]
ok so my third recording would be set for Gamine Mode, would be 1080 mhz

**Comment by SkippTekk** [comment]
oh, do you want me to put "logging" checked and upload the files?

**Comment by SkippTekk** [comment]
Video 2 https://www.youtube.com/watch?v=GOTTS_WTu3I&feature=youtu.be

**Comment by jackun** [comment]
Pretty blocky. For now, all I can suggest is to try with even lower GPU clocks as just yesterday i did some FO4 caps on R9 290 and seemed fine to me. I'll check if there's some wierd corner cases later.

**Comment by SkippTekk** [comment]
alright man, but i still wanna know if you want me to look into the Log check, should i start logging for you?

**Comment by jackun** [comment]
Should have no need, for now atleast.

**Comment by SkippTekk** [comment]
alright man, i have the 1080 video recorded and rendered, just uploading the video now. by the way. em i keeping you up with this? cause i hope im not. it's currently turning 4 am my time right now :v

**Comment by jackun** [comment]
No 1PM here, go to sleep ;P

**Comment by SkippTekk** [comment]
nah man, i work late shifts. so it's normal for me to be up this late :)

**Comment by SkippTekk** [comment]
Test 3 video uploaded https://www.youtube.com/watch?v=NK5D50mU8QM&feature=youtu.be

**Comment by SkippTekk** [comment]
i just realised. would being in the GPU beta have a huge effect to this?

**Comment by jackun** [comment]
MSI gaming app? Dunno, it shouldn't unless it does some automatic background clocking. Or set clocks from AMD Catalyst control panel.

**Comment by SkippTekk** [comment]
well no, i was on the BETA 15.11 and i just installed 15.7.1 back onto my machine. that could help the thing cause it's more stable?

**Comment by jackun** [comment]
Oh right, well, never say never, but I'm on 15.11 too and seemed fine.

**Comment by SkippTekk** [comment]
alright....that worked? the video looks WAY smoother now with this recent recording. im rendering it now.

**Comment by SkippTekk** [comment]
Test 5? https://youtu.be/RWEFdWz8OH8

**Comment by SkippTekk** [comment]
would it be better if i would to set the Profile to High, and the Quality at about 15 instead of 20? like this setting? 

https://i.gyazo.com/ae779b52f56f2435599893cebec46c91.png

**Comment by jackun** [comment]
You recorded at 60 fps? Set the FPS override in codec config to 60 / 1 and check `Enabled`. Unless mplayer plays frame-by-frame wrong, the YT 60 fps video seems to be 30 instead.

CQP value is kind of a matter of taste and for what purpose you record. The usual quality/file size trade-off. If going by default settings AMF limits QP to 18..51. But high profile is probably preferable.

**Comment by SkippTekk** [comment]
alright man! thanks for all the help, but it's time for me to take off and head to bed. Your help is very much appreaciated and here is the latest video i recorded and it went smooth to my understanding! 

Things go Boom: https://www.youtube.com/watch?v=aX28lDHnHmc&feature=youtu.be

**Comment by SkippTekk** [comment]
alright, just did some more tests. and i noticed that the audio isn't in sync.
